Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,16 +1,28 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
 using System.IO;
-using System.Linq;
-using System.Text;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
+        {   
+            //part 1
+            FileCleaner.Clean(@"Input\Balances.corrupted", @"Input\Balances.clean");
+
+            Part2();
+        }
+
+        static void Part2()
         {
+            var fileContent = File.ReadAllText(@"Input\Balances.clean");
+            var peopleAndBalances = fileContent.Split(Environment.NewLine);
+
+            //    BalanceStats.ConvertToBalanceArray(peopleAndBalances[0]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5012948)